api/v1alpha3/awsmachine_webhook.go

```diff
@@ -78,14 +91,33 @@ func (r *AWSMachine) ValidateUpdate(old runtime.Object) error {
 	delete(oldAWSMachineSpec, "additionalSecurityGroups")
 	delete(newAWSMachineSpec, "additionalSecurityGroups")
 
+	// allow changes to secretARN
```

Comment:
Enabled cannot be changed, but secretARN can?

Reply:
To echo @vincepri what is the situation in which we'd expect a user to update the `secretARN`?

Reply:
Yes. Controller can update SecretARN as it's generated. But there's no use changing the value of "Encrypted" after the fact.